### ISSUE TITLE
Cannot start local generator instance

### ISSUE THREAD

**Issue by pixels4nickels** [adobe-photoshop/generator-core#388]
When trying to start a local node instance running generator, I am met with this:
`node app.js --debug -v -f ../plugins
[info:core 13:06:18.166 generator.js:103:17] Launching with config:
{}
Communication error: 2
[warning:core 13:06:18.181 generator.js:131:25] photoshop communications error: {"error":"Photoshop communication error: 2"}
Communication error: 2
[warning:core 13:06:18.182 generator.js:131:25] photoshop communications error: {"error":"Photoshop communication error: 2"}
[info:core 13:06:18.184 generator.js:117:25] Photoshop connection closed
Exiting with code 0: Generator close event
`
I am running Node 4.4.0. I am blocked on all development for the updated Apple Layered Image Exporter plugin due to a needed fix in 3.1.1 of generator-core, which has still not been released.

I filed against the issue(about 9 months ago) that was fixed in 3.11 but it was removed from the GitHub issue system..Also the commit where it was fixed by an Adobe developer has also been removed, making it appear the issue never existed! Why?!?! I was never notified it was fixed, and it is STILL broken in CC2017, even though the code has been fixed in the repo...

https://github.com/adobe-photoshop/generator-core/commit/192be7e789f1a787cda891b48bbb9e0f61c16733#diff-8317c373a0e944920aed59122ba0938b

Line 216 *actually originally read as "log.log()" but was corrected to say "logger.log()".
So what happened to this commit? I still see:
`[0m[1;31mExiting with code -1: Uncaught exception: Cannot read property 'log' of undefined`
In my OS X console logs when trying to use Flite in the most up to date Photoshop.

**Comment by mcilroyc** [comment]
@pixels4nickels the master branch of generator will not run on Photoshop 2017.  See [Forum Post](https://groups.google.com/forum/#!topic/adobe-generator-dev/ztHCYaIvlrE).  That is the cause of your "Communication Error" noted above.  

I do not understand the second half of your issue report.  Generator version 3.8.4 is included in PS 2017.  Are you saying there is a bug in that version that has since been fixed?

**Comment by pixels4nickels** [comment]
Thanks for that @mcilroyc! That will help me continue development...whew.
The previous issue I spoke of was 9 months ago and has been fixed in the codebase, yet it still happens in CC2017. The logger argument at one time was referred to as log and caused an exception. It is still happening even though I see in 3.11 the usage of the logger argument was corrected. It is really, really strange. If I set useFlite to true, it blows up on that same logging statement. Thanks so much for the quick response!
